master-protected-branches/master.cfg

```diff
@@ -297,17 +297,6 @@ c['builders'].append(
       properties={'mtr_additional_args': protected_branches_mtr_additional_args},
       factory=f_quick_build))
 
-c['builders'].append(
-    util.BuilderConfig(name="amd64-fedora-35",
-      workernames=workers["x64-bbw-docker-fedora-35"],
-      tags=["Fedora", "quick", "gcc"],
-      collapseRequests=True,
-      nextBuild=nextBuild,
-      canStartBuild=canStartBuild,
-      locks=getLocks,
-      properties={'mtr_additional_args': protected_branches_mtr_additional_args},
-      factory=f_quick_build))
-
```

Comment:
if we want fedora36 to replace fedora35 as a protected branches builder, this needs to be updated to use fedora36 (see also comment above)